### PR TITLE
VB-1815, added word-break to all govuk-table cells, to fix long string issue

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
## Description
On active alerts, when a long string with no spaces had been added in comments, the string continued across the page, breaking out of its box and pushing all other content right, introducing side scrolling.

`overflow-wrap: anywhere`  is being used as initial suggestion had been deprecated

### CODE REVIEW

File: assets/sass/local.scss [hunk 1]
```diff
@@ -247,3 +247,7 @@
     margin-bottom: govuk-spacing(3);
   }
 }
+
+.govuk-table__cell {
+  word-break: break-word;
+}
```
Comment (psoleckimoj) [inline assets/sass/local.scss:253]: Looks good, just need a newline leaving at the end of the file 👍

**Review by psoleckimoj — DISMISSED** [review]
LGTM!

File: assets/sass/local.scss [hunk 2]
```diff
@@ -247,3 +247,7 @@
     margin-bottom: govuk-spacing(3);
   }
 }
+
+.govuk-table__cell {
+ overflow-wrap: anywhere;
```
Comment (tpmcgowan) [inline assets/sass/local.scss:252]: Tried this out locally and it's causing words to break mid-way through where it shouldn't happen (see screenshot). Depends a lot on window size so you don't always see it. Might need to be more targeted to the specific columns/cells likely to contain this kind of data?


Screenshot 2023-02-10 at 08 10 40

**Review by tpmcgowan — CHANGES_REQUESTED** [review]
I think this needs tweaking so it doesn't cause 'real' words to break in other cells (see comment with screenshot)

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**